Forms/ConfigPanels/UCNames.cs

```diff
@@ -442,8 +530,16 @@ private void ToolStripButtonLoad_Click(object sender, EventArgs e)
 
             m_NameList.Clear();
             bool bResult = m_NameList.Load();
-            PopulateList();
+            try
+            {
+                PopulateList();
+            }
+            catch (NullReferenceException)
+            {
+                // todo

```

Comment:
same as above, another todo for a caught error - does this need to be handled?

Reply:
that one doesn't and I found how to correct the first one, so no need to catch the error and it load without conflict